pareto/strategic_water_management/strategic_produced_water_optimization.py

```diff
@@ -3402,6 +3488,454 @@ def SeismicActivityExceptionRule(model, k, t):
     return model
 
 
+def operational_hydraulics(model, opt):
```

Comment:
Given that this is the strategic model, not the operational model, I think that the function name `operational_hydraulics` is potentially confusing and I suggest changing it to something else.

Reply:
updated it to call the method pipeline_hydraulics